tests/std/tests/P0896R4_ranges_alg_uninitialized_fill/test.cpp

```diff
@@ -141,6 +141,20 @@ struct throwing_test {
     }
 };
 
+struct memset_test {
+    static constexpr int expected[] = {42, 42, 42};

```

Comment:
This isn't exercising `memset`. We have tests above that do: `// Validate int is properly converted to bool` tests filling 1-byte elements (`bool`) with an arbitrary value (`5` converted to `true`), and `// Validate zero-ing` tests filling 4-byte `int` with `0`.

The `int` `0` test is basically comprehensive (and would have caught the bug here), but `bool` is weird, so should we replace this new test with a 1-byte elements, like filling `unsigned char` with `42`?